Crust Crawler Arms/read_joint_positions/read_joint_positions.ino

```diff
@@ -34,8 +34,10 @@ void loop(){
   // and send it to the PC.
   for(int i = 1; i < 6; ++i){
     int pos = GetPosition(i);
-    String report;
-    report = i + ": " + pos;
+    String servo=String(i);
+    String text=String(": ");
+    String posit=String(pos);
+    String report = String(servo+text+posit);
```

Comment:
One more thing, I might have mislead you. The IDE can compile, but if I upload the code to the robot and open the serial monitor with the original code I only get a bunch of ones. Nothing else. However changing the code seemed to work.